thunder/torch/__init__.py

```diff
@@ -3875,8 +3878,17 @@ def pad_lo_hi_dilation_seq():
     padding, a = process_padding_str(int_to_seq(padding), stride, dilation, a)
     # }
 
-    res = clang.convolution(
-        a, weight, bias, stride, padding, dilation, False, (0,) * dim, groups  # transposed  # output_padding
+    res = conv_function(
+        a,
```

Comment:
nit: looks like `conv_function == _convolution_autocast_impl` accepts kwargs here, but `_convolution_autocast_impl` misses kwargs in its definition.

Reply:
Yes, but this is to be able to route dtype through without burdening the conv helper to know about dtypes.